src/runtime_src/driver/xclng/drm/xocl/xocl_drv.h

```diff
@@ -111,15 +111,22 @@ static inline bool uuid_is_null(const xuid_t *uuid)
 #define	XOCL_CHARDEV_REG_COUNT	16
 
 #ifdef RHEL_RELEASE_VERSION
-#if RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(7,3)
-#define RHEL_P2P_SUPPORT  1
-#else
-#define RHEL_P2P_SUPPORT  0
+
+#if RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(7,6)
+#define RHEL_P2P_SUPPORT_74  0
+#define RHEL_P2P_SUPPORT_76  1
+#elif RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(7,3) && RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(7,6)
+#define RHEL_P2P_SUPPORT_74  1
+#define RHEL_P2P_SUPPORT_76  0
 #endif
 #else
-#define RHEL_P2P_SUPPORT  0
+#define RHEL_P2P_SUPPORT_74  0
+#define RHEL_P2P_SUPPORT_76  0
 #endif
 
+
+#define RHEL_P2P_SUPPORT (RHEL_P2P_SUPPORT_74 | RHEL_P2P_SUPPORT_76)
```

Comment:
Just curious: what is the motivation for testing for some distribution instead of just looking at the kernel version or a kernel feature?

Reply:
Instead of upgrading the whole kernel (and bump up kernel version), RedHat/CentOS keep backporting individual fixes from latest kernel without bumping up their kernel version. Ubuntu will just upgrade the entire kernel (and bump up kernel version).